docs/diagrams/notes/NotesCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
+
+participant "AddNoteCommandParser" as Parser
+participant "ParserUtil" as Util
+participant "AddNoteCommand" as Command
+participant "Model" as Model
+participant "Person" as Person
+participant "Note" as Note
+
+Parser -> Util : parseIndex(args[0])
+activate Util
+Util --> Parser : index: Index
```

Comment:
not sure if this annotation is really allowed? safer to just put `index` as the returned result or even just omitting it